js2xml/__main__.py

```diff
@@ -2,6 +2,7 @@
 import sys
 from argparse import ArgumentParser
 
+import six
 import js2xml
```

Comment:
There should be an empty line between these two imports, as per PEP8. Sorry I missed this in my first review.

Reply:
Thanks, added.